connect/runtime/src/main/java/org/apache/kafka/connect/storage/KafkaConfigStorage.java

```diff
@@ -301,7 +301,7 @@ public void putConnectorConfig(String connector, Map<String, String> properties)
      */
     public void putTaskConfigs(Map<ConnectorTaskId, Map<String, String>> configs) {
         // Make sure we're at the end of the log. We should be the only writer, but we want to make sure we don't have
-        // any outstanding lagging data to consume.
+        // any outstanding logging data to consume.
```

Comment:
lagging is actually the correct word here. We are consuming messages that are outstanding because the consumer is lagging behind the latest data in the broker.